ikasaneip/component/endpoint/amazon-s3-endpoint/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,36 @@
+# SLF4J's SimpleLogger configuration file
```

Comment:
this seems like new method of defining logs.   does it actually work

Reply:
Yes this does work - think the solr client uses this as well. Can see this output if you change level to INFO :- 

```
ain] INFO org.ikasan.component.endpoint.amazon.s3.client.AmazonS3Client - About to upload byte array of size [11] bytes with key [hello-world.txt] to Amazon S3 bucket [ikasan-test-bucket]
[main] INFO org.ikasan.component.endpoint.amazon.s3.client.TransferManagerProgressLogger - Uploading to ikasan-test-bucket/hello-world.txt :   [####################]

Process finished with exit code 0

```